src/utils/string.ts

```diff
@@ -55,7 +55,7 @@ export const sanitize = (
   },
 ) => {
   const { whitespace = '', underscore = '', dot = '', dash = '' } = options || {};
-  let newValue = value.replace(/[^\w\s.-]/g, '');
```

Comment:
don't need to add the _  was already work like that normally

Reply:
Indeed, TIL `underscore` is included in `\w`, my bad